alf/examples/ppo_procgen/base_conf.py

```diff
@@ -83,8 +82,8 @@ def value_network_ctor(input_tensor_spec):
 alf.config(
     'TrainerConfig',
     unroll_length=256,
-    mini_batch_length=1,
-    mini_batch_size=4096,
+    mini_batch_length=256,
```

Comment:
batch length from the replay buffer is actually 257, so this will ignore the last step. It can be set to `None` so that min_batch_length is equal to the length from replay buffer

Reply:
Yes this holds true if `keep_last_exp` is turned on. However, in current `alf` it is not turned on. But the `None` trick should be the most adaptive option. Thank you!

https://github.com/HorizonRobotics/alf/blob/breakds_exp/alf/algorithms/algorithm.py#L1198 Confirmed that, and updated here with `None` with documentation explaining that.